include/ginkgo/core/base/std_extensions.hpp

```diff
@@ -27,8 +27,24 @@ namespace gko {
  * @ingroup xstd
  */
 namespace xstd {
+namespace detail {
+
+
```

Comment:
Maybe add some documentation on what the intended use is.

Reply:
There is no use for users for this. On the developer side it's used directly below, so I don't think it requires documentation.